### DISCUSSION TITLE
Custom CSS with light and dark themes

### DISCUSSION THREAD

**Discussion by SavArmadillo** [directus/directus#13142] [Archive]
Hello!

I am quite new with Directus and I am currently trying to create some custom interfaces with the help of the built-in Project settings custom CSS section.

Currently I have created a custom CSS variable in my interface CSS. 
Let's call it `--background-interface-test`.

I can now use that variable to change the background color of my interface directly in the project settings : 
![image](https://user-images.githubusercontent.com/91469954/166945490-12ac3931-bec3-463c-8cc1-fee2303c3fe9.png)

However I can't figure out how to adapt that color depending of the current user theme mode (light or dark).

Is there a way to do it?

Thanks in advance!

**Comment by azrikahar** [comment]
You should be able to use `light`/`dark` class on the body when a user uses one of them specifically, and `@media (prefers-color-scheme: dark)`/`@media (prefers-color-scheme: light)` for `auto` 👍 Here's a quick clip for illustration purpose:

https://user-images.githubusercontent.com/42867097/166949521-5715b1db-6b86-4fe3-85f0-5f87d2ebaac2.mp4

**Reply by SavArmadillo** [reply]
Thanks! I managed to do it with your quick response:

![image](https://user-images.githubusercontent.com/91469954/166951938-8dbf10a9-2c96-49a9-9aaa-d81d4748b67e.png)

I look forward to improve my skills on this app, it seems very interesting to use!
